apps/pokedex/pokedex.star

```diff
@@ -0,0 +1,54 @@
+"""
+Applet: Pokedex
+Summary: Display a random Pokemon
+Description: Display a random Pokemon along with its height and weight.
+Author: Mack Ward
+"""
+
+load("http.star", "http")
+load("render.star", "render")
+load("schema.star", "schema")
+load("time.star", "time")
+
+NUM_POKEMON = 386
+POKEAPI_URL = "https://pokeapi.co/api/v2/pokemon/{}"
```

Comment:
A few things we've learned to check, just wanted to document that we have checked them:
- The SSL cert is provided by cloudflare so we shouldn't have to worry about it expiring
- Looks like there are very limited rules and don't require attribution: https://pokeapi.co/docs/v2
- This API appears to be statically hosted by cloudflare, so I don't suspect it will go down very frequently 😄